_data/blogs.yml

```diff
@@ -262,3 +262,8 @@
   title: Explore & Discover a new essence in Technology
   img: Saniya blog pic.jpg
   author_name: Saniya Kolangde
+  
+- url:https://mygeekyworld786.wordpress.com/
+  title: Here I come Coding In
+  img: haider2
```

Comment:
You need to write  the name of the image file that you upload on the `img/blog/` folder. Also, add the image in the said folder.